rocky/reports/report_types/mail_report/report.py

```diff
@@ -46,44 +46,36 @@ def generate_data(self, input_ooi: str, valid_time: datetime) -> Dict[str, Any]:
         number_of_dkim = number_of_hostnames
 
         for hostname in hostnames:
-            measures = self._get_measures(valid_time, hostname.reference)
-            mail_security_measures[hostname.primary_key] = measures
+            finding_types[hostname.primary_key] = self._get_mail_finding_types(valid_time, hostname.reference)
 
             number_of_spf -= (
                 1
-                if list(
-                    filter(lambda finding: finding.id == "KAT-NO-SPF", mail_security_measures[hostname.primary_key])
-                )
+                if list(filter(lambda finding: finding.id == "KAT-NO-SPF", finding_types[hostname.primary_key]))
```

Comment:
`finding` should also be renamed to `finding_type` here and in the other three lambdas below. I can't do this with a suggestion in github because black will also reformat the code when the line length is longer again.